test/test_operations.py

```diff
@@ -674,6 +674,14 @@ def test_norm_p0(self):
     xla_norm = xla_a.norm(p = 0)
     self.assertEqual(norm, xla_norm)
 
+  def test_slice_start_end(self):
```

Comment:
We should use `runAtenTest()` as it reduces the amount of code:

https://github.com/pytorch/xla/blob/5067d0e34c06cf8a437942686daa48b0be62c0f4/test/test_operations.py#L719